src/components/Popup.js

```diff
@@ -21,6 +26,16 @@ export default class Popup {
     }
   };
 
+  uploadingInfo(isloading) {
```

Comment:
`uploadingInfo` should be only in `PopupWithForm.js` because `Popup` is used for the image modal and it doesn't have any submit button